source/integrations/index.html

```diff
@@ -131,185 +105,169 @@
+            <div class='title'>${component.title}</div>
+          </a>
+          `).join('\n');
+      } else {
+        rendered = '<p class="note">Nothing found!</p>';
+      }
 
       // set active class on active menu item
-      jQuery('.filter-button-group a.active').removeClass('active');
-      jQuery(`.filter-button-group a[href="${hash}"]`).addClass('active');
+      document.querySelector('.filter-button-group a.active')?.classList?.remove?.('active');
+      document.querySelector(`.filter-button-group a[href="${hash}"]`)?.classList?.add?.('active');
       if (hash === "") {
-        jQuery('.filter-button-group a[href*="#featured"]').addClass('active');
+        document.querySelector('.filter-button-group a[href*="#featured"]').classList.add('active');
       }
 
       // remove previous elements and css classes, add the new stuff and then trigger the fade-in css animation
-      $('#componentContainer').html('').removeClass('show-items remove-items').html(rendered).addClass('show-items');
-      logoLazyLoad.update();
+      componentContainerEl.innerHTML = '';
+      componentContainerEl.classList.remove('show-items');
+      componentContainerEl.classList.remove('remove-items');
+      componentContainerEl.innerHTML = rendered;
+      componentContainerEl.classList.add('show-items');
     }
 
-    /**
-     * update the browser location hash. This enables users to use the browser-history
-     */
-    function updateHash(newHash) {
-      if ('pushState' in history) {
-        history.pushState('', '', newHash);
-      } else {
-        location.hash = newHash;
-      }
+    function buildImageEl(component) {
+      const urlBase = [
+        'https://brands.home-assistant.io',
+        component.ha_brand ? 'brands' : '_',
+        component.domain,
+        'logo'
+      ].join('/');
+
+      return `<img src="${urlBase}.png" srcset="${urlBase}@2x.png 2x" loading="lazy">`;
     }
 
     // update view by filter selection
-    jQuery('.filter-button-group a').click(function () {
-      updateHash(this.getAttribute('href'));
-      applyFilter();
-
-      return false;
-    });
+    for (let filterLink of document.querySelectorAll('.filter-button-group a')) {
+      filterLink.addEventListener('click', () => {
+        history.pushState('', '', filterLink.getAttribute('href'));
+        applyFilter();
+        return false;
+      });
+    }
 
     // update view on select change
-    jQuery('select').change(function () {
-      updateHash(this.value);
+    const versionsEl = document.getElementById('versions');
+    versionsEl.addEventListener('change', () => {
+      history.pushState('', '', versionsEl.value);
       applyFilter();
-
-      return false;
     });
 
     /**
-     * Simple debounce implementation, based on http://davidwalsh.name/javascript-debounce-function
+     * Simple debounce implementation
      */
-    function debounce(func, wait, immediate) {
```

Comment:
For the use case on this page:
 - `immediate` was never removed - deleted.
 - `func` never has any arguments - removed arg handling